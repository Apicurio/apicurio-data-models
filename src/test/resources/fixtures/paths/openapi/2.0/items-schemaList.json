{
    "openapi": "2.0.0",
    "info": {
        "title": "Temporary API",
        "version": "1.0.0",
        "description": ""
    },
    "definitions": {
        "ItemsTest1": {
            "description": "",
            "type": "array",
            "items": {
                "format": "date",
                "type": "string"
            }
        },
        "ItemsTest2": {
            "description": "",
            "type": "array",
            "items": [
                {
                    "format": "date",
                    "type": "string"
                },
                {
                    "type": "boolean"
                },
                {
                    "type": "object",
                    "properties": {
                        "name": {
                            "type": "string"
                        },
                        "id": {
                            "format": "int32",
                            "type": "integer"
                        }
                    }
                }
            ]
        }
    }
}
