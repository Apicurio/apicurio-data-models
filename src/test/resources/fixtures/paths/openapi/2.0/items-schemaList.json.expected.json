{
    "type": "object",
    "properties": {
        "name": {
            "type": "string"
        },
        "id": {
            "format": "int32",
            "type": "integer"
        }
    }
}
